_includes/header.html

```diff
@@ -85,6 +85,14 @@
 					</a>
 				</li>
 			{% endif %}
+
+			{% if page.url contains '/spec/v1/components/' %}
+				<li>
+					<a href="/spec/v1/components/" {% if page.url == '/spec/v1/components/' %}aria-current="true"{% endif %}>
```

Comment:
Not Important (nit): I was reading about aria-current and found that we can improve our use of it by setting the value to `page` instead of `true`.

This is the page I was reading -- https://www.aditus.io/aria/aria-current/#aria-current-page